TestTransfers/file_format_extensions_only/README.md

```diff
@@ -0,0 +1,24 @@
+## Variously configured files to test handling of file format extensions
+
+The files in these folders are designed to test various different format
+identification tools' (and the format policy register's) ability to look at
```

Comment:
Should "register" be "registry" here?

Reply:
Also, are you talking about the tools' ability to "look at" or their ability to "identify"?

Reply:
Super nitpicky, but I notice that neither of your dirs contain a `_six` file? Did you miss one you originally intended to put in? Superstition?

Reply:
Ah! `_six` that old number! We don't need that one!!! ;D 

(I had accidentally chosen one on `.gitignore`, `.pyc` as it was a good one for testing against. Will figure out a different extension.